build/ps-entrypoint.sh

```diff
@@ -226,7 +226,7 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 		echo 'Initializing database'
 		# we initialize database into $TMPDIR because "--initialize-insecure" option does not work if directory is not empty
 		# in some cases storage driver creates unremovable artifacts (see K8SPXC-286), so $DATADIR cleanup is not possible
-		"$@" --initialize-insecure --skip-ssl --datadir="$TMPDIR"
+		"$@" --initialize-insecure --datadir="$TMPDIR"
```

Comment:
does it work if user disables tls when creating a new cluster?

Reply:
but we do not support it 
https://github.com/percona/percona-server-mysql-operator/blob/main/deploy/cr.yaml#L9-L14
and if it does not work we will not have it for now

Reply:
true sorry confused with pxc